src/pages/leejiyon/Main/Components/Feeds.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Article from './Article';
+
+class Feeds extends Component {
+  state = {
+    articleList: [],
+  };
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/leejiyon/articledata.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          articleList: data,
+        });
+      });
```

Comment:
어떤 data인지 구체적으로 써주시면 좋을 것 같아요~!
```suggestion
      .then(articleData => {
        this.setState({
          articleList: articleData,
        });
      });
```

Reply:
확인했습니다